javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
@blakeembrey not sure the performance is the issue here, but `instanceof` is much more performant, see [the benchmark](https://jsperf.com/instanceof-vs-object-prototype-tostring2).